README.md

```diff
@@ -1,194 +1,64 @@
+A lot of things have changed since version 2 of the Apify SDK, including the split of the crawlers to the new [`crawlee`](https://npmjs.org/crawlee) module.
+But fear not, as we've written a guide to help you easily migrate from v2 to v3! Visit the [Upgrading Guide](https://crawlee.dev/docs/upgrading/upgrading-to-v3)
+to find out what changes you need to make (especially the section related to this very [Actor SDK](https://crawlee.dev/docs/upgrading/upgrading-to-v3#actor-sdk)),
+and, if you encounter any issues, join our [Discord server](https://discord.gg/jyEM2PRvMU) for help!
 
 ## Quick Start
 
 This short tutorial will set you up to start using Apify SDK in a minute or two.
-If you want to learn more, proceed to the [Getting Started](https://sdk.apify.com/docs/guides/getting-started)
-tutorial that will take you step by step through creating your first scraper.
+If you want to learn more, proceed to the [Apify Platform](https://crawlee.dev/docs/guides/apify-platform)
+guide that will take you step by step through running your actor on Apify's platform.
 
-### Local stand-alone usage
+### Basic Example
 
-Apify SDK requires [Node.js](https://nodejs.org/en/) 15.10 or later.
-Add Apify SDK to any Node.js project by running:
+Actor SDK requires [Node.js](https://nodejs.org/en/) 16 or later.
+Add Actor SDK to any Node.js project by running:
 
 ```bash
-npm install apify playwright
+npm install apify crawlee playwright
 ```
 
-> Neither `playwright` nor `puppeteer` are bundled with the SDK to reduce install size and allow greater
-> flexibility. That's why we install it with NPM. You can choose one, both, or neither.
-
-Run the following example to perform a recursive crawl of a website using Playwright. For more examples showcasing various features of the Apify SDK,
-[see the Examples section of the documentation](https://sdk.apify.com/docs/examples/crawl-multiple-urls).
-
-```javascript
-const Apify = require('apify');
+> For this example, we'll also install the [`crawlee`](https://npmjs.org/crawlee) module, as it now provides the crawlers that were previously exported
+> by the Apify SDK. If you don't plan on writing crawlers in your actors, then you don't need to install it!
+> Keep in mind that neither `playwright` nor `puppeteer` are bundled with `crawlee` in order to reduce install size and allow greater
+> flexibility. That's why we manually install it with NPM. You can choose one, both, or neither.
 
-// Apify.main is a helper function, you don't need to use it.
-Apify.main(async () => {
-    const requestQueue = await Apify.openRequestQueue();
-    // Choose the first URL to open.
-    await requestQueue.addRequest({ url: 'https://www.iana.org/' });
+```typescript
+import { Actor } from 'apify';
+import { PlaywrightCrawler } from 'crawlee';
 
-    const crawler = new Apify.PlaywrightCrawler({
-        requestQueue,
-        handlePageFunction: async ({ request, page }) => {
+await Actor.main(async () => {
```

Comment:
We should probably prefer the `init/exit` way instead of `Actor.main`. I'd be in favour of having both examples, as people are familiar with the `main` method, but with `init/exit` you dont need to nest anything which is better for readability.